ADR/ADR014-idp-authentication.md

```diff
@@ -0,0 +1,25 @@
+# ADR014: Use an Identity Provider for authentication and implement authorisation
+
+Date: 2023-02-06
+
+## Status
+
+Accepted
+
+## Context
+
+In [ADR003 Use GOV.UK Signon](ADR003-use-govuk-signon.md) we recognised that
+
+> We don't yet know if/how self-service account creation could work
+
+Self-service account creation is essential to allow adoption to scale in public beta. This is not going to be possible with GOV.UK Signon so we need to find an alternative way to authenticate our users.
```

Comment:
I'm not sure how much detail we include in these but is it worth saying very briefly why it's not possible to use Signon for self/service account creation? Is it because Signon accounts have to be created by a GDS admin?

Reply:
Thanks for your comments @hannahkc @bettymw - I've added some detail.